### PR TITLE
Adding Python wrapper for antenna checking (ant) and Python regressio…

### PR DESCRIPTION
**Pull request by macd**
…n tests

Signed-off-by: Don MacMillen <don@macmillen.net>

### CODE REVIEW

File: src/ant/test/check_api1.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ack.checkAntennas("", False)
+count = ack.antennaViolationCount()
+print(f"violation count = {count}", flush=True)
+
+# note that the tcl function check_net_violations is a boolean but
+# the tcl reg test treats it as an int. So we print an int here
+# so we can have the same log file
+net = "net50"
+viol = 1 if ant.anyViolations(net) else 0
+print(f"Net {net} violations: {viol}")
```
Comment (maliberty) [inline src/ant/test/check_api1.py:21]: Can you just do
```
print(f"Net {net} violations: {int(viol)}")
```

File: src/ant/src/AntennaChecker-py.i [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+{ 
+  odb::dbNet* net = ord::getDb()->getChip()->getBlock()->findNet(net_name);
+  if (net) {
+    auto antChecker = ord::OpenRoad::openRoad()->getAntennaChecker();
+    auto vios = antChecker->getAntennaViolations(net, nullptr);
+    return !vios.empty();
+  }
+  else
+    return false;
+}
```
Comment (maliberty) [inline src/ant/src/AntennaChecker-py.i:69]: Do we need this or could you just call 
```
antChecker->getAntennaViolations(net, nullptr);
```
directly from python

Comment (maliberty) [reply src/ant/src/AntennaChecker-py.i]: I added
```
%template(ViolationVector) std::vector<ant::Violation>;
```
to AntennaChecker-py.i and in the test used
```
net = design.getBlock().findNet('net50')
ack.getAntennaViolations(net, None);
```
without a problem.

**Comment by macd** [comment]
looks good.  I'll do that.
Don

**Review by maliberty — APPROVED** [review]

**Pull request merged**